Resources/Prototypes/Roles/Jobs/Engineering/chief_engineer.yml

```diff
@@ -3,6 +3,13 @@
   name: job-name-ce
   description: job-description-ce
   playTimeTracker: JobChiefEngineer
+  requirements:
+    - !type:CharacterTraitRequirement
+      inverted: true
+      traits:
+        - Foreigner
+        - ForeignerLight
+        - Blindness
```

Comment:
Blindness needs to be taken out of every role but Security/HoS (Shockingly, fighting is not strictly a part of the Captain's job description). Otherwise once that's done it should be good to merge.